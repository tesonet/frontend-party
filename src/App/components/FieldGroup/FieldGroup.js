import React from 'react';
import styled from 'styled-components';
import { FormGroup, FormControl } from 'react-bootstrap';

const FieldGroup = ({ ...props }) => {
  return (
    <FormGroup>
      <InputField {...props} />
    </FormGroup>
  )
}

const InputField = styled(FormControl)`
  height: 45px;

  &:focus {
    color: #999;
  }
`;

export default FieldGroup;
