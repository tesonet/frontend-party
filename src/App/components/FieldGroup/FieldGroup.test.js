import React from 'react';
import renderer from 'react-test-renderer';
import FieldGroup from './FieldGroup';

it('renders correctly', () => {
  const FielGroupComponent = renderer
    .create(<FieldGroup />)
    .toJSON();
  expect(FielGroupComponent).toMatchSnapshot();
});