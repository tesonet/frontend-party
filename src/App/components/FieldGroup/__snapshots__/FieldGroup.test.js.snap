// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form-group"
>
  <input
    className="sc-bdVaJa injyga form-control"
  />
</div>
`;
